src/GongSolutions.WPF.DragDrop/DragDrop.Properties.cs

```diff
@@ -1148,5 +1148,30 @@ public static void SetRootElementFinder(UIElement target, IRootElementFinder val
         {
             target.SetValue(RootElementFinderProperty, value);
         }
+
+        /// <summary>
+        /// Gets or sets the handler for the dragged items sorter
+        /// </summary>
+        public static readonly DependencyProperty DragPreviewSortHandlerProperty
+            = DependencyProperty.RegisterAttached("DragPreviewSortHandler",
```

Comment:
@Deadpikle As you already mentioned, maybe this should be called `DragPreviewItemsSorter`